packages/entity/src/utils/testing/__tests__/StubDatabaseAdapter-test.ts

```diff
@@ -192,6 +192,81 @@ describe(StubDatabaseAdapter, () => {
       expect(results).toHaveLength(3);
       expect(results.map((e) => e.stringField)).toEqual(['c', 'b', 'a']);
     });
+
+    it('supports null field values', async () => {
+      const queryContext = instance(mock(EntityQueryContext));
+      const databaseAdapter = new StubDatabaseAdapter<TestFields>(
+        testEntityConfiguration,
+        StubDatabaseAdapter.convertFieldObjectsToDataStore(
+          testEntityConfiguration,
+          new Map([
+            [
+              testEntityConfiguration.tableName,
+              [
+                {
+                  customIdField: '1',
+                  testIndexedField: 'h1',
+                  numberField: 1,
+                  stringField: 'a',
+                  dateField: new Date(),
+                  nullableField: 'a',
+                },
+                {
+                  customIdField: '2',
+                  testIndexedField: 'h2',
+                  numberField: 2,
+                  stringField: 'a',
+                  dateField: new Date(),
+                  nullableField: 'b',
+                },
+                {
+                  customIdField: '3',
+                  testIndexedField: 'h3',
+                  numberField: 3,
+                  stringField: 'a',
+                  dateField: new Date(),
+                  nullableField: null,
+                },
+                {
+                  customIdField: '3',
+                  testIndexedField: 'h3',
+                  numberField: 3,
```

Comment:
Make this row distinguishable from the row above:
```suggestion
                  customIdField: '4',
                  testIndexedField: 'h4',
                  numberField: 4,
```